ChangeLog.rst

```diff
@@ -0,0 +1,62 @@
+Shoop Change Log
```

Comment:
It's kind of "standard" that ChangeLog is in repository root. Maybe we could symlink it to doc though, but let's check that before the next release. OK?